### PR TITLE
schema name support for tables names

### PR DESCRIPTION
**Pull request by soforth**
1. add scheme name support for tables names
2. add IF EXIST support for DROP TABLE/VIEW
3. fix memory free bug: delete -> free
4. add features in sqlhelper.cpp

### CODE REVIEW

**Comment by mrzzzrm** [comment]
Hi, @soforth, thanks for contributing! Could you maybe add some now supported queries to `test/queries/queries_good.sql`?

File: src/parser/bison_parser.y [hunk 1]
```diff
@@ -108,7 +108,8 @@ int yyerror(YYLTYPE* llocp, SQLParserResult* result, yyscan_t scanner, const cha
 	hsql::PrepareStatement* prep_stmt;
 	hsql::ExecuteStatement* exec_stmt;
 	hsql::ShowStatement*    show_stmt;
-
+	
+	hsql::TableName table_n;
```
Comment (mrzzzrm) [inline src/parser/bison_parser.y]: I think `table_name` would be a better name for this.

**Comment by soforth** [comment]
hi @mrzzzrm modifications are made as you advised, please revise again, thanks.

**Comment by soforth** [comment]
@mrzzzrm This is just a minimal modification,  I 'd planned to expand the column data types to solve the limitations mentioned in the docs/known-limitations.md. Just as what I did in my own go version sql-parser (https://github.com/soforth/sql-parser) when I was free.  I am not sure whether it would be welcomed or not.

**Comment by mrzzzrm** [comment]
I've made some changes to this PR and reopened it in #70.

**Pull request closed without merging**